cloudformation/database/serverless-database.yaml

```diff
@@ -268,7 +268,7 @@ Resources:
     Properties:
       DBClusterIdentifier: !Ref ClusterName
       Engine: "aurora-mysql"
-      EngineVersion: "8.0.mysql_aurora.3.03.0"
+      EngineVersion: "8.0.mysql_aurora.3.07.0"
```

Comment:
what about setting EngineVersion to 8.0 rather than specifying the precise patch version? In this way  it will use the latest version by default and we do not need to make this change once again. See [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#cfn-rds-dbcluster-engineversion).

Reply:
Done